apps/backoffice-v2/src/common/components/molecules/ProcessTracker/hooks/useProcessTrackerLogic/process-tracker-adapters/process-tracker.abstract.ts

```diff
@@ -0,0 +1,17 @@
+import { TPlugin } from '@/domains/workflow-definitions/fetchers';
+import { TWorkflowById } from '@/domains/workflows/fetchers';
+
+export interface ProcessTrackerItem {
+  text: string | JSX.Element | undefined;
+  leftIcon: JSX.Element | undefined;
```

Comment:
Using `ReactNode` tends to be more flexible and accurate. i.e. passing a span with text over just a string and so on.
```suggestion
  text: ReactNode | ReactNode[];
  leftIcon: ReactNode | ReactNode[];
```